test/jest.ts

```diff
@@ -0,0 +1,152 @@
+import { describe, expect, test } from '@jest/globals';
+export { expect } from '@jest/globals';
+import { dirname } from 'path';
+import {
+    createProgram,
+    findConfigFile,
+    flattenDiagnosticMessageText,
+    getPreEmitDiagnostics,
+    parseJsonConfigFileContent,
+    readConfigFile,
+    sys
+} from 'typescript';
+
+export type Test = () => void;
+
+export function runTests(
+    canisterName: string,
+    tests: Test,
+    cwd: string = process.cwd()
+) {
+    const { shouldRunTests, shouldRunTypeChecks, shouldRunBenchmark } =
+        processEnvVars();
+    if (shouldRunTests) {
```

Comment:
spaces between these kinds of statements

Reply:
done